instance/models/log_entry.py

```diff
@@ -21,24 +21,22 @@
 """
 
 # Imports #####################################################################
-
 import logging
 
+from django.contrib.contenttypes.models import ContentType
 from django.db import models
+from django.db.models.signals import post_delete
 from django_extensions.db.models import TimeStampedModel
 
-from instance.models.instance import SingleVMOpenEdXInstance
-from instance.models.server import OpenStackServer
-from instance.models.utils import ValidateModelMixin
-
+from .utils import ValidateModelMixin
 
 # Logging #####################################################################
 
 logger = logging.getLogger(__name__)
 
-
```

Comment:
@bradenmacdonald Nit: We should keep this blank link to stay consistent with the formatting in other files.